frontend/src/store/reportChanges.js

```diff
@@ -310,7 +310,10 @@ export const useReportChangesStore = defineStore('reportChanges', {
       let payload = {
         applicationId: applicationStore.applicationId,
         programYearId: applicationStore.programYearId,
-        providerType: organizationStore.organizationProviderType == 'GROUP' ? 100000000 : 100000001,
+        providerType:
+          organizationStore.organizationProviderType === ORGANIZATION_PROVIDER_TYPES.GROUP
+            ? ORGANIZATION_PROVIDER_TYPES_IDS.GROUP
+            : ORGANIZATION_PROVIDER_TYPES_IDS.FAMILY,
```

Comment:
I think this same code is in organization.js. It might make a good utility method in organizationMixin.js.

Reply:
on it !